examples/Example_embed_shift_template_experiment.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
<p>sorry what is background here, just other samples in the compendium? otherwise really nice!! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/ponyo/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Yes "background" refers to the background training compendium used. I'll update this to be "background compendium" instead